README.md

```diff
@@ -7,6 +7,19 @@ Leverage the client-server architecture of [Kakoune] to connect programs to clie
 [![connect.kak](https://img.youtube.com/vi_webp/jca2N-cE_mM/maxresdefault.webp)](https://youtube.com/playlist?list=PLdr-HcjEDx_k-Y_9uSV0YAUCNHzqHjmz3 "YouTube – connect.kak")
 [![YouTube Play Button](https://www.iconfinder.com/icons/317714/download/png/16)](https://youtube.com/playlist?list=PLdr-HcjEDx_k-Y_9uSV0YAUCNHzqHjmz3) · [connect.kak](https://youtube.com/playlist?list=PLdr-HcjEDx_k-Y_9uSV0YAUCNHzqHjmz3)
 
+The objective of connect.kak is
+to sinchronize external applications with kakoune clients easily.
+A typical use case is opening a file browser and
+having it open the files in the kakoune client.
+Another very typical use case is connecting a terminal.
+
+connect.kak provides basic [commands] to interact with the connected client interactively
+or to write your own scripts
+(check [`:batch`][:batch], which is an integration crafted from the rest of the commands)
```

Comment:
Can you simplify the reference as ``[`:batch`]``?

Reply:
I had changed it (though I haven't pushed yet) to "batch.kak" and a link to alexherbo2/bath.kak, instead of the source. I can link to the source in the plugin, and display it as ` ``:batch` `` then

Reply:
ouki

When done, could you squash the commits into a single one?

Reply:
> ouki
> 
> When done, could you squash the commits into a single one?

Sure, I remember you prefer it squashed.

Reply:
Done